engine/flink/avro-util/src/test/scala/pl/touk/nussknacker/engine/avro/sink/KafkaAvroSinkFactoryValidationSpec.scala

```diff
@@ -0,0 +1,92 @@
+package pl.touk.nussknacker.engine.avro.sink
+
+import com.typesafe.config.ConfigFactory
+import org.apache.flink.api.scala._
+import org.scalatest.{FunSuite, Matchers}
+import pl.touk.nussknacker.engine.Interpreter
+import pl.touk.nussknacker.engine.api.context.ProcessCompilationError.{CustomNodeError, NodeId}
+import pl.touk.nussknacker.engine.api.context.ValidationContext
+import pl.touk.nussknacker.engine.api.namespaces.DefaultObjectNaming
+import pl.touk.nussknacker.engine.api.process.ProcessObjectDependencies
+import pl.touk.nussknacker.engine.api.{MetaData, StreamMetaData}
+import pl.touk.nussknacker.engine.avro.KafkaAvroFactory.{SchemaVersionParamName, SinkOutputParamName, TopicParamName}
+import pl.touk.nussknacker.engine.avro.schemaregistry.confluent.ConfluentSchemaRegistryProvider
+import pl.touk.nussknacker.engine.compile.ExpressionCompiler
+import pl.touk.nussknacker.engine.compile.nodevalidation.{GenericNodeTransformationValidator, TransformationResult}
+import pl.touk.nussknacker.engine.expression.ExpressionEvaluator
+import pl.touk.nussknacker.engine.graph.evaluatedparam.Parameter
+import pl.touk.nussknacker.engine.graph.expression.Expression
+import pl.touk.nussknacker.engine.spel.Implicits._
+import pl.touk.nussknacker.engine.testing.{EmptyProcessConfigCreator, LocalModelData}
+
+import scala.collection.JavaConverters._
+
+class KafkaAvroSinkFactoryValidationSpec extends FunSuite with Matchers with KafkaAvroSinkSpecMixin {
```

Comment:
Why not in `KafkaAvroSinkFactorySpec.scala`?

Reply:
True, merged